proposals/csharp-8.0/default-interface-methods.md

```diff
@@ -109,23 +109,23 @@ interface IA
 }
 interface IB : IA
 {
-    override void IA.M() { WriteLine("IB.M"); } // explicitly named
+    void IA.M() { WriteLine("IB.M"); } // Explicit implementation
 }
 interface IC : IA
 {
-    override void M() { WriteLine("IC.M"); } // implicitly named
+    void M() { WriteLine("IB.M"); } // Creates a new M, unrelated to `IA.M`. Warning
 }
 ```
 
-Override declarations in interfaces may not be declared `sealed`.
+Explicit implementations in interfaces may not be declared `sealed`.
 
-Public `virtual` function members in an interface may be overridden in a derived interface explicitly (by qualifying the name in the override declaration with the interface type that originally declared the method, and omitting an access modifier).
+Public `virtual` function members in an interface may be implemented in a derived interface explicitly (by qualifying the name in the override declaration with the interface type that originally declared the method, and omitting an access modifier).
 
-`virtual` function members in an interface may only be overridden explicitly (not implicitly) in derived interfaces, and members that are not `public` may only be implemented in a class or struct explicitly (not implicitly). In either case, the overridden or implemented member must be *accessible* where it is overridden.
+`virtual` function members in an interface may only be implemented explicitly (not implicitly) in derived interfaces, and members that are not `public` may only be implemented in a class or struct explicitly (not implicitly). In either case, the overridden or implemented member must be *accessible* where it is overridden.
```

Comment:
It looks like there is still a reference to "overridden" term in this paragraph.